modules/statics/src/account.ts

```diff
@@ -265,6 +265,38 @@ export class AvaxERC20Token extends ContractAddressDefinedToken {
   }
 }
 
+/**
+ * FIAT based coins, such as USD, EUR, or YEN.
+ */
+export class FiatCoin extends BaseCoin {
```

Comment:
Can we rename to `FiatToken` so we don't have a mismatch with the name of the factory function?

Reply:
Done!